azure-resources/DesktopVirtualization/hostPools/recommendations.yaml

```diff
@@ -42,7 +42,7 @@
   recommendationResourceType: Microsoft.DesktopVirtualization/hostPools
   recommendationMetadataState: Active
   longDescription: |
-    For optimized AVD configuration, place Hybrid VMs in unique OUs. Segregate Prod and DR units for environment-specific settings. This ensures targeted configurations for session hosts, including FSLogix, timeouts, and session controls.
+    Place domain joined session hosts VMs in unique OUs. Segregate Prod and DR units for environment-specific settings. This ensures targeted configurations for session hosts, including FSLogix, session controls, etc.
```

Comment:
For clarity, small rewrite:

Segregate Prod and DR units into unique Organizational Units (OU) for environment-specific settings, ensuring targeted configurations for session hosts including FSLogix, session controls, etc.